eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -65,6 +65,9 @@ jobs:
         VCPKG_DEFAULT_TRIPLET: 'x64-windows-static'
         Package.EnableSBOMSigning: true
       steps:
+        - template: /eng/common/pipelines/templates/steps/credscan.yml
```

Comment:
Is it OK if this runs in CI in public?

Reply:
The credscan team page does not limit us for internal use. The log did not expose our internal information out. We currently have this setup in public ci for all other 4 langs. It is good to go.